spec/features/navigation/navigation_spec.rb

```diff
@@ -76,8 +76,48 @@
     expect(page).to_not have_link("Login")
     expect(page).to_not have_link("Register")
   end
+
+  it "shows different things in the navbar for admin" do
+
+    admin_1 = User.create!(
+      name: "Jaffar",
+      street_address: "789 Palace Street",
+      city: "Detroit",
+      state: "AZ",
+      zip: 98345,
+      email: "jaffar@gamil.com",
+      password: "geniessuck",
+      role: 3
+     )
+
+    visit '/login'
+
+    fill_in :email, with: "jaffar@gamil.com"
+    fill_in :password, with: "geniessuck"
+
+    click_button "Log In"
+save_and_open_page
+    expect(page).to have_link("Admin Dashboard")
```

Comment:
Admin nav bar spec looking for certain links to show and not show up according to story 5